certbot/renewal.py

```diff
@@ -304,6 +304,8 @@ def _renew_describe_results(config, renew_successes, renew_failures,
         notify(report(renew_skipped, "skipped"))
     if not renew_successes and not renew_failures:
         notify("No renewals were attempted.")
+        if config.pre_hook is not None or config.post_hook is not None:
```

Comment:
In my mind they are a little different, but that's probably just me. I'll send a follow up commit so we're consistent here.